MoltenVK/MoltenVK/GPUObjects/MVKQueue.mm

```diff
@@ -218,8 +218,9 @@
 
 void MVKQueue::initName() {
 	const char* fmt = "MoltenVKQueue-%d-%d-%.1f";
-	char name[256];
-	sprintf(name, fmt, _queueFamily->getIndex(), _index, _priority);
+	size_t nameLen = 256;
```

Comment:
This should probably be a `const`; otherwise, you're needlessly creating a VLA. Or, you could pass `sizeof(name)/sizeof(char)` to `snprintf(3)`.

Reply:
Thanks. Good catch. Fixed now.